### ISSUE TITLE
When bootstrapping, connect to a peer only once concurrently

### ISSUE THREAD

**Issue by rkeene** [nanocurrency/nano-node#1522]
When bootstrapping, connect to a peer only once concurrently -- to avoid having a client connect to the same peer as all of its connections

**Comment by SergiySW** [comment]
Closed with https://github.com/nanocurrency/nano-node/pull/1590
Can be removed from 18 milestone & project as duplicate of PR

**Comment by SergiySW** [comment]
Closed with #1590
